webapps/frontend/public/app/admin/locales/de.json

```
@@ -82,6 +82,7 @@
     "AUTH_LOGOUT_SUCCESSFUL": "Sie haben sich erfolgreich abgemeldet",
     "AUTH_LOGOUT_THANKS": "Vielen Dank für die Nutzung von Camunda. Wir wünschen einen schönen {{dayContext}}!",
     "BACK": "Zurück",
+    "BANNER_CE_EOL_TEXT": "Probiere doch mal Camunda 8 aus! Das letzte Release von Camunda 7 CE wird im Oktober 2025 sein. Wenn du Fragen dazu hast, kontaktiere uns im a href=\"https://forum.camunda.io/t/important-update-camunda-7-community-edition-end-of-life-announced/50921\" target=\"_blank\">Forum</a>.",
```

🔧 Fix opening link tag

```suggestion
    "BANNER_CE_EOL_TEXT": "Probiere doch mal Camunda 8 aus! Das letzte Release von Camunda 7 CE wird im Oktober 2025 sein. Wenn du Fragen dazu hast, kontaktiere uns im <a href=\"https://forum.camunda.io/t/important-update-camunda-7-community-edition-end-of-life-announced/50921\" target=\"_blank\">Forum</a>.",
```